android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/JsonExtension.kt

```diff
@@ -31,6 +32,10 @@ val json = Json {
 inline fun <reified T> String.decodeJson(jsonInstance: Json? = null): T =
   jsonInstance?.decodeFromString(this) ?: json.decodeFromString(this)
 
+/** Decode string to an entity of type [T] */
```

Comment:
can you please expand the doc string here to have at least an @return and @param following the multiline format here, https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax

(if you could also do that for the pre-existing line function on line 40, that'd be much appreciated :D)